src/azure-cli/azure/cli/command_modules/acs/_client_factory.py

```diff
@@ -59,8 +59,9 @@ def get_auth_management_client(cli_ctx, scope=None, **_):
 
 def get_container_service_client(cli_ctx, **_):
     from azure.mgmt.containerservice import ContainerServiceClient
+    from azure.cli.core.profiles import ResourceType, get_api_version
 
-    return get_mgmt_service_client(cli_ctx, ContainerServiceClient)
+    return get_mgmt_service_client(cli_ctx, ContainerServiceClient, api_version=get_api_version(cli_ctx, ResourceType.MGMT_CONTAINERSERVICE))
```

Comment:
formatting, i believe CI pyblinter will flag this

Reply:
Fixed